internal/mosaic/command/builder.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 func Build(m mosaic.Mosaic, cfg *config.Config) []string {
-	playlistPath := fmt.Sprintf("hls/%s/playlist.m3u8", m.Name)
+	playlistPath := fmt.Sprintf("hls/%s/playlist-%s.m3u8", m.Name, m.Name)
```

Comment:
Why repeating the name of the mosaic again?

Reply:
`hls/mosaicname/playlist.m3u8` 

This is how the path looks like. It works for HLS.

Reply:
To make unique each mosaic configured in tasks.json, because if you create a new mosaic object and maintains a playlist.m3u8 name, the final result will be mixed into a unique HLS output, therefore into HLS player, I can to change the manifest.

Reply:
Now I get what you are saying. But the answer for this problem is creating a new bucket. Every mosaic should have its own bucket, then, for example:
* mosaic1 will have its files in mosaic1 bucket
* mosaic2 will have its files in mosaic2 bucket

Reply:
I'm not sure when the bucket should be created. Probably just before starting the ffmpeg process

Reply:
Or we can to create only one bucket, like `hls` or `objects` and use the prefixes to organize fragments, see how this works in AWS S3 [article](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-prefixes.html).
What do you think?

Reply:
Look this: 
![bucket with directories](https://github.com/mauricioabreu/mosaic-video/assets/34068867/647110f6-3460-4062-839f-56c5e9f12964)

Basically, join the folder name + filename and upload object:
![image](https://github.com/mauricioabreu/mosaic-video/assets/34068867/626aa568-21c1-4e57-a834-e4371b44f543)


This way we'll have one bucket organized by prefixes.

Reply:
That looks great!